src/syntaxtree.c

```diff
@@ -343,6 +497,43 @@ static Obj SyntaxTreeFunc(Obj result, Obj func)
     return result;
 }
 
+static UInt SyntaxTreeCodeFunc_Internal(Obj node)
+{
+    Int narg = INT_INTOBJ(ElmPRec(node, RNamName("narg")));
+    Int nloc = INT_INTOBJ(ElmPRec(node, RNamName("nloc")));
+    Obj nams = ElmPRec(node, RNamName("nams"));
+    CodeFuncExprBegin(narg, nloc, nams, 0);
+    Obj  stat_rec = ElmPRec(node, RNamName("stats"));
+    Obj  body_stats = ElmPRec(stat_rec, RNamName("statements"));
+    UInt nr_stats = LEN_PLIST(body_stats);
+    for (int i = 1; i <= nr_stats; i++) {
+        Expr current = SyntaxTreeDefaultCoder(ELM_PLIST(body_stats, i));
+        PushStat(current);
+    }
+    return nr_stats;
+}
+
+static Expr SyntaxTreeCodeFunc(Obj node)
+{
+    UInt nr_stats = SyntaxTreeCodeFunc_Internal(node);
+    CodeFuncExprEnd(nr_stats,1);
```

Comment:
```suggestion
    CodeFuncExprEnd(nr_stats, 1);
```
Maybe run clang-format

Reply:
Done